phpBB/memberlist.php

```diff
@@ -1448,6 +1448,20 @@
 				usort($user_list,  'phpbb_sort_last_active');
 			}
 
+			// do we need to display contact fields as such
+			$use_contact_fields = false;
+
+			/**
+			 * Modify the query used to get the users for the team page
```

Comment:
yes but it doesn't allow you to edit the sql query itself but if yes or no the contact fields should be used